packages/sdk-core/src/events.ts

```diff
@@ -77,6 +77,7 @@ export interface FlowUpdatedEvent extends EventBase {
     flowRate: string;
     flowOperator: string;
     deposit: string;
+    streamId: string; // Subgraph ID.
```

Comment:
I would personally be a bit more explicit here and say something like `Subgraph Superfluid Stream ID`.
When I see Subgraph ID, I initially think of this (might just be me though): `QmVQcG7NusjRarbmL4717WoaaAx8mSynTrek73NmznYYp3` the IPFS hash of the deployed subgraph.

Reply:
Changed: 
<img width="960" alt="image" src="https://user-images.githubusercontent.com/10894666/216268983-83d1acf3-efc5-40a1-8a52-5aa981af4543.png">

Reply:
even better!